openslides/users/models.py

```diff
@@ -14,20 +17,96 @@
 from openslides.utils.models import AbsoluteUrlMixin
 from openslides.utils.rest_api import RESTModelMixin
 
+from .exceptions import UserError
+
 
 class UserManager(BaseUserManager):
     """
     UserManager that creates new users only with a password and a username.
     """
-
     def create_user(self, username, password, **kwargs):
         user = self.model(username=username, **kwargs)
         user.set_password(password)
         user.save(using=self._db)
         return user
 
+    def create_or_reset_admin_user(self):
+        """
+        Creates an user with the username admin. If such a user exists, resets
+        it. The password is (re)set to 'admin'. The user becomes member of the
+        group 'Staff' (pk=4).
+        """
+        try:
+            admin = self.get(username='admin')
+        except self.model.DoesNotExist:
+            admin = self.model()
+            admin.username = 'admin'
+            admin.last_name = 'Administrator'
+            created = True
+        else:
+            created = False
```

Comment:
proposal:
```python
admin, created = self.get_or_create(username='admin', defaults={'last_name: 'Administrator'})
```
The disadvantage is, that a second save is called, when the admin is created. But the code is much more clearer and when this command is called, speed is not so essential.